### PR TITLE
Finished leave group and display joined members in Chat Settings View

### PR DESCRIPTION
**Pull request by JustinLovesCompsci**

### CODE REVIEW

File: SwiftParseChat/ChatViewController.swift [hunk 1]
```diff
@@ -348,7 +348,6 @@ class ChatViewController: JSQMessagesViewController, UICollectionViewDataSource,
         if segue.identifier == "PushToSettingsSegue" {
             let createVC = segue.destinationViewController as ChatSettingsViewController
             createVC.groupId = self.groupId
-            createVC.users = self.users
```
Comment (huyouare) [inline SwiftParseChat/ChatViewController.swift:351]: Was there a downside to this approach? 
I feel like making fewer network requests is better.

**Comment by huyouare** [comment]
Other than that note in ChatVC, looks good to merge.

**Comment by huyouare** [comment]
Other than that comment, looks good to merge.

File: SwiftParseChat/ChatViewController.swift (hunk 1 above)
Comment (JustinLovesCompsci) [inline SwiftParseChat/ChatViewController.swift:351]: I used this before but didn't work. Users array in chat view may contain duplicate users cuz it put a user in for every message, not reversely. If a user has not posted any message, he won't be in the array

**Comment by JustinLovesCompsci** [comment]
Re-implement this class with fewer network requests. Deleted this merge request

**Comment by JustinLovesCompsci** [comment]
Added a group field in class. Deleted extra network request while leaving group.

**Comment by huyouare** [comment]
LGTM.

**Pull request merged**